inference-engine/src/offline_transformations/src/pruning/pruning.cpp

```diff
@@ -15,8 +15,11 @@ NGRAPH_RTTI_DEFINITION(ngraph::pass::Pruning, "Pruning", 0);
 
 bool ngraph::pass::Pruning::run_on_function(std::shared_ptr<Function> f) {
     Manager manager(get_pass_config());
+
+    manager.register_pass<InitMasks>();
```

Comment:
I would add some comment that we actually initialize masks during propagation but for specific cases we do it as a separate pass.

Reply:
Added